### ISSUE TITLE
Add check if user is in kvm group

### ISSUE THREAD

**Issue by nanu-c** [ubports/ubports-pdk#17]
The pull command shouldn't work if the user isn't in the kvm group. The priority is low because I run it outside of snap.
```
nanu@Maulwurf:/media/nanu/home2/src/ut/ubports-pdk$ ./ubuntu-touch-pdk pull
#####################################
Ubuntu Touch Platform Development Kit
#####################################

Executing tasks:
	Setup? no
	Clear cache? no
	List? no
	Pull image? yes
	Run an image? no

--2021-07-11 20:56:55--  https://ci.ubports.com/job/Platform%20Development%20Kit/job/pdk-vm-image-amd64/lastSuccessfulBuild/artifact/ubuntu-touch-pdk-amd64.raw.xz
Auflösen des Hostnamens ci.ubports.com (ci.ubports.com) … 172.67.150.210, 104.21.0.97, 2606:4700:3033::ac43:96d2, ...
Verbindungsaufbau zu ci.ubports.com (ci.ubports.com)|172.67.150.210|:443 … verbunden.
HTTP-Anforderung gesendet, auf Antwort wird gewartet … 200 OK
Länge: 718382184 (685M) [application/x-xz]
Wird in ‘/media/nanu/home2/src/ut/pdk/pdk-image-cache/default/ubuntu-touch-pdk-amd64.raw.xz’ gespeichert.

ubuntu-touch-pdk-amd64.raw.xz                       100%[=================================================================================================================>] 685,10M  15,4MB/s    in 45s     

2021-07-11 20:57:40 (15,4 MB/s) - ‘/media/nanu/home2/src/ut/pdk/pdk-image-cache/default/ubuntu-touch-pdk-amd64.raw.xz’ gespeichert [718382184/718382184]

Unpacking the archive
Pull successful!
nanu@Maulwurf:/media/nanu/home2/src/ut/ubports-pdk$ ./ubuntu-touch-pdk run
#####################################
Ubuntu Touch Platform Development Kit
#####################################

Executing tasks:
	Setup? no
	Clear cache? no
	List? no
	Pull image? no
	Run an image? yes


Name of the environment: default

qemu-system-x86_64: warning: dbind: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-WrX2pMLhwJ: No such file or directory
```

**Comment by nanu-c** [comment]
Okey even setup has to be rerun, else i get something like 

```
nanu@Maulwurf:/media/nanu/home2/src/ut/ubports-pdk$ qemu-ut-pdk.qemu-virgil -device virtio-keyboard-pci -device virtio-mouse-pci -enable-kvm -device virtio-vga,virgl=on -display gtk,gl=on -cpu Haswell-v4 -netdev user,id=ethernet.0 -device rtl8139,netdev=ethernet.0 -device AC97 -serial mon:stdio -m 8G -smp 6 -drive if=virtio,format=raw,file=/media/nanu/home2/src/ut/pdk/pdk-image-cache/default/hdd.raw,discard=on -drive if=virtio,format=raw,file=/media/nanu/home2/src/ut/pdk/sshd/settings.raw,readonly=o
qemu-system-x86_64: warning: dbind: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-WrX2pMLhwJ: No such file or directory

(qemu-system-x86_64:65672): Gtk-WARNING **: 21:31:52.661: Theme parsing error: gtk.css:1565:23: 'font-feature-settings' is not a valid property name

(qemu-system-x86_64:65672): Gtk-WARNING **: 21:31:52.665: Theme parsing error: gtk.css:3617:25: 'font-feature-settings' is not a valid property name

(qemu-system-x86_64:65672): Gtk-WARNING **: 21:31:52.666: Theme parsing error: gtk.css:4079:23: 'font-feature-settings' is not a valid property name
qemu-system-x86_64: -drive if=virtio,format=raw,file=/media/nanu/home2/src/ut/pdk/pdk-image-cache/default/hdd.raw,discard=on: Could not open '/media/nanu/home2/src/ut/pdk/pdk-image-cache/default/hdd.raw': Permission denied

```

**Comment by fredldotme** [comment]
You need to understand and follow the Snap's permission system and connect the `removable-media` plug:
`sudo snap connect ubports-pdk:removable-media`
or:
`sudo snap connect qemu-ut-pdk:removable-media`

**Comment by fredldotme** [comment]
Tbf, we could add a check and ask users to add themselves to the group manually, but teh README already contains info about that.

**Comment by fredldotme** [comment]
Implemented as of: 35556414a84c93a0d66c7c38dfe1d5bd9d6dc1c3

This is *only* implemented for the full-Snap PDK. Please check the `--edge` channel for another release test (or build the snap yourself using `snapcraft`) in an hour from now.

**Comment by fredldotme** [comment]
Seems to have fixed it. Closed.
